fenix/app/src/test/java/org/mozilla/fenix/home/HomeFragmentTest.kt

```diff
@@ -123,4 +138,42 @@ class HomeFragmentTest {
 
         assertFalse(homeFragment.shouldEnableWallpaper())
     }
+
+    @Test
+    fun `GIVEN removed a pinned top site WHEN showUndoSnackbarForTopSite is called THEN show the correct snackbar`() {
```

Comment:
```suggestion
    fun `WHEN a pinned top is removed THEN show the undo snackbar`() {
```

We prefer to use plain english and avoid using the actual code names since those could easily be refactored and render the test description to be completely out of date.